oauthenticator/oauth2.py

```diff
@@ -494,14 +517,22 @@ def _validate_server_cert_default(self):
         help="""
         Extra default kwargs passed to all HTTPRequests.
 
-        For example, to use a HTTP proxy for all requests:
+        .. code-block:: python
 
-        `c.OAuthenticator.http_request_kwargs = {"proxy_host": "proxy.example.com", "proxy_port": 8080}`
+            # Example: send requests through a proxy
+            c.OAuthenticator.http_request_kwargs = {
+                "proxy_host": "proxy.example.com",
+                "proxy_port": 8080,
+            }
 
-        See the `tornado.httpclient.HTTPRequest` documentation for all options and limitations:
-        https://www.tornadoweb.org/en/stable/httpclient.html#tornado.httpclient.HTTPRequest
+            # Example: validate against certain root certificates
+            c.OAuthenticator.http_request_kwargs = {
+                "ca_certs": "/run/secrets/kubernetes.io/serviceaccount/ca.crt",
```

Comment:
```suggestion
                "ca_certs": "/path/to/a.crt",
```
Makes it clearer this is an example- or is there a reason to use the K8s CA?

Reply:
Seems like it can be a relevant use case as its the default if the file is found for the OpenShiftOAuthenticator, but its a specific example so we can go with a non-specific example like this!